### PR TITLE
Luexradio json data

### PR DESCRIPTION
**Pull request by cp-shruti-s**

### CODE REVIEW

File: vue-frontend/src/pages/portfolio/[id].vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+export default {
+  data() {
+    return {
+      id: this.$route.params.id,
+    };
+  },
+  components: {
+    Header,
+    LandingSection,
```
Comment (cp-sumi-k) [inline vue-frontend/src/pages/portfolio/[id].vue:20]: I think the Landing section is part of another ticket, We have to add logic that how we load json data based on the parameter of the URL.

File: vue-frontend/src/components/portfolio/LandingSection.vue [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<template>
+  <div>
+    <div>{{ jsonData.title }}</div>
+    <div>{{ jsonData.subTitle }}</div>
+    <img :src="jsonData.backgroundImage" />
+  </div>
+</template>
+
+<script>
+import luxeradioJsonData from "@/luxeradio-data.json";
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/LandingSection.vue:10]: JSON data is dynamic for the landing section, we can not directly assign luxeradio data to this section

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/portfolio/LandingSection.vue (hunk 2 above)
Comment (cp-shruti-s) [reply vue-frontend/src/components/portfolio/LandingSection.vue:10]: I know, that's why add that if condition in `mounted hook` and assign only `luxeradioJsonData.details.landingSection`

File: vue-frontend/src/pages/portfolio/[id].vue (hunk 1 above)
Comment (cp-shruti-s) [reply vue-frontend/src/pages/portfolio/[id].vue:20]: Add that section only reference purpose

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**